numpy/linalg/linalg.py

```diff
@@ -930,7 +930,11 @@ def eigvalsh(a, UPLO='L'):
         computed.
     UPLO : {'L', 'U'}, optional
         Same as `lower`, with 'L' for lower and 'U' for upper triangular.
-        Deprecated.
+        Deprecated. 
```

Comment:
Hmm, this doesn't look deprecated in the code. @pv Is this a stray bit left over from the merge of the gufuncs?